### PR TITLE
Convert more exercises

### PR DESCRIPTION
**Pull request by nerdinand**

### CODE REVIEW

File: exercises/4/todo-list.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+<b>Finish Rubymonstas exercise</b>
+Please enter your command!
+<b>quit</b>
+Goodbye!{% endcapture %}
+
+    {% include exercises/program_output.html program_output=program_output %}
+  </li>
+</ol>
+
+
```
Comment (hharen) [inline exercises/4/todo-list.md]: Add hint: Implement means make it work?

File: exercises/5/methods.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      end
+
+      f
+    end
+    ```
+
+    First, try to understand step by step what this method does.
+
+    Here comes the challenge: Can you write a recursive method that calculates the factorial?
+
```
Comment (hharen) [inline exercises/5/methods.md]: Can you write a recursive method that calculates the factorial? That means a method that will call itself to calculate it?

File: exercises/5/methods.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    ```
+
+    What does the `mystery` method return when we call it with:
+
+    1. `mystery(3)`
+
+    2. `mystery(4)`
+
+    Try to figure this out first without running the code. Just think about what happens, line after line! If you like, you can use the [Schreibtischtest](https://de.wikipedia.org/wiki/Schreibtischtest) strategy for this.
+
```
[the remark this answers is not in the document]
Comment (nerdinand) [reply exercises/5/methods.md]: There isn't really an english word for it (the WP article links to https://en.wikipedia.org/wiki/Dry_run_(testing)) which doesn't seem like the exact same concept to me and the article is way shorter.

Will suggest that, good point.

File: exercises/4/todo-list.md (hunk 1 above)
Comment (nerdinand) [reply exercises/4/todo-list.md]: You mean it isn't clear what "implement" means? In that case I'd probably rather replace it with a different word. What do you think?

File: exercises/5/methods.md (hunk 2 above)
Comment (nerdinand) [reply exercises/5/methods.md]: Yes, I'll add some more hints to make this clearer...

File: exercises/4/todo-list.md (hunk 1 above)
Comment (hharen) [reply exercises/4/todo-list.md]: Yeah, it's often not clear to the beginners. But I'd leave it there with an explanation. Like this, they'll gradually learn also the programming jargon :)

Comment (nerdinand) [reply exercises/4/todo-list.md]: Is something like this understandable enough?

```markdown
__Hint__: In programming, we use the verb *implement* a lot, e.g. *implement this feature*. It just means "to write code that has the specified feature when executed".
```

I struggle a bit with finding a good, understandable definition...

Comment (hharen) [reply exercises/4/todo-list.md]: :DDD well, it's a bit better? ;oP 

why not to write:

In programming, we use the verb *implement* a lot, e.g. *implement this feature*. It just means make it work. Write the code which will make it happen.

btw. execute is another such word, I should finish the dictionary I started to write ;oP

**Pull request merged**